server/common/admin-rest-client/src/main/java/org/kaaproject/kaa/server/common/admin/AdminClient.java

```diff
@@ -1062,26 +1062,25 @@ public UserDto editUserProfile(UserDto userDto) {
         return response.getBody();
     }
 
-    public CredentialsDto provisionCredentials (String applicationId, String credentials) {
+    public CredentialsDto provideCredentials (String applicationId, String credentialsBody) {
```

Comment:
I believe "provision" is correct. By definition, provision is the act of providing or supplying something. This was the original name of the method.